src/AzureAuth/CommandMain.cs

```diff
@@ -99,6 +100,12 @@ public CommandMain(CommandExecuteEventData eventData, ILogger<CommandMain> logge
         [Option(TenantOption, "The ID of the Tenant where the client and resource entities exist in", CommandOptionType.SingleValue)]
         public string Tenant { get; set; }
 
+        /// <summary>
+        /// Gets or sets the customized header text for WAM prompts.

```

Comment:
```suggestion
        /// Gets or sets the customized header text for WAM prompts and web.
```
for consistency with line 106.

Reply:
Thank you. I am reviewing. Since these commits contains lots changes, I converted this PR to a draft.

Reply:
> Thank you. I am reviewing. Since these commits contains lots changes, I converted this PR to a draft.

Got it! sorry didn't notice the draft mode.

Reply:
It's ready for review now ; )